airbyte-workers/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -126,11 +129,14 @@ private int computeExitValue() {
       // we read the status from the Kubernetes API, we need to check the doc store again.
       final AsyncKubePodStatus secondDocStoreStatus = getDocStoreStatus();
       if (secondDocStoreStatus.equals(AsyncKubePodStatus.FAILED)) {
+        log.info("State Store reports orchestrator pod {} failed", getInfo().name());
         return 1;
       } else if (secondDocStoreStatus.equals(AsyncKubePodStatus.SUCCEEDED)) {
+        log.info("State Store reports orchestrator pod {} succeeded", getInfo().name());
         return 0;
       } else {
         // otherwise, the actual pod is terminal when the doc store says it shouldn't be.
+        log.info("State Store missing status, however orchestrator pod {} in terminal. Assume failure.", getInfo().name());
```

Comment:
@davinchia Same comment:  this seems like it should be error, not info (or at lest warn).